components/ExploreLayoutWithSidebar.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div
+    class="container is-fluid"
+    :class="{ 'sidebar-padding-left': isSidebarOpen }"
+  >
+    <slot />
+  </div>
+</template>
+
+<script lang="ts" setup>
+// convert to nested layout:  explore-layout-with-sidebar -> explore-layout
+const { getsidebarFilterCollapse: isSidebarOpen } = storeToRefs(usePreferencesStore())
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/abstracts/variables';
+
+.sidebar-padding-left {
+  padding-left: 0;
+}
+
+// this cover the edge case where sidebar is open and then screen size changes to mobile
+// for exmpale on rotation of tablet device
```

Comment:
nit: I think it's quite rare that user rotate on tablet device. even our visitor is quite rare from tablet. for comparison from our cf-analytics
desktop: 4k
mobile: 3k
tablet: 10 (yep, just ten)

so, I think we can safe to delete this kind of stuff